### PR TITLE
Build update for android

### PR DESCRIPTION
**Pull request by jbygdell**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [X] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:
Add a variables file that contains the versions of Qt, SDK and NDK as well as the other variables we use to build the mobile app for android. 

### Changes made:
1). Add variables file
2). Update the build scripts to source the variables file.
3). Update the INSTALL file to reflect the changes made.

### CODE REVIEW

**Comment by neolit123** [comment]
lgtm and the build system is happy too.
comments by others?

File: packaging/android/build.sh [hunk 1]
```diff
@@ -90,14 +90,8 @@ elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.9" ] ; then
 	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.9
 elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.8" ] ; then
 	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.8
-elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.7" ] ; then
-	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.7
-elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.6" ] ; then
-	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.6
-elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.5" ] ; then
-	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.5
```
Comment (dirkhh) [inline packaging/android/build.sh:98]: would it be a good idea to mention that you removed support for Qt5.5-5.7

**Review by dirkhh — CHANGES_REQUESTED** [review]
I like it - it would be nice to mention the removal of Qt 5.5-5.7 (I completely agree with it, it's just worth mentioning, I think)

File: packaging/android/build.sh (hunk 1 above)
Comment (jbygdell) [reply packaging/android/build.sh:98]: Commit message updated.

**Review by glance- — CHANGES_REQUESTED** [review]

**Review by dirkhh — APPROVED** [review]

File: packaging/android/android-build-wrapper.sh [hunk 2]
```diff
@@ -70,24 +63,33 @@ if [ ! -d $ANDROID_NDK ] ; then
...
-	# FIXME: Read these from build.sh varables, or install them there.
-	tools/bin/sdkmanager tools platform-tools 'platforms;android-27' 'build-tools;25.0.3'
 	popd
 fi
 
 # download the Qt installer including Android bits and unpack / install
 QT_DOWNLOAD_URL=https://download.qt.io/archive/qt/${QT_VERSION}/${LATEST_QT}/${QT_BINARIES}
-if [ ! -d Qt ] ; then
+if [ ! -d Qt/${LATEST_QT}/android_armv7 ] ; then
+	if [[ -d Qt ]] ; then
```
[the remark this answers is not in the document]
Comment (jbygdell) [reply packaging/android/android-build-wrapper.sh]: Probably sticky fingers in combination with auto closing feature of the IDE, fixed during rebase.

**Review by glance- — APPROVED** [review]

**Comment by dirkhh** [comment]
Thanks

**Pull request merged**